cmd/spire-server/cli/run/run.go

```diff
@@ -525,6 +549,67 @@ func NewServerConfig(c *Config, logOptions []log.Option, allowUnknownConfig bool
 	return sc, nil
 }
 
+func parseBundleEndpointProfile(config federatesWithConfig) (trustDomainConfig *bundleClient.TrustDomainConfig, err error) {
+	var data bytes.Buffer
+	if err := printer.DefaultConfig.Fprint(&data, config.BundleEndpointProfile); err != nil {
+		return nil, err
+	}
+	configString := data.String()
+	profileConfig := new(bundleEndpointProfileConfig)
+	if err := hcl.Decode(profileConfig, configString); err != nil {
+		return nil, fmt.Errorf("failed to decode configuration: %w", err)
+	}
+
+	var endpointProfile bundleClient.EndpointProfileInfo
+	switch {
+	case profileConfig.HTTPSWeb != nil:
+		endpointProfile = bundleClient.HTTPSWebProfile{}
+	case profileConfig.HTTPSSPIFFE != nil:
+		spiffeID, err := spiffeid.FromString(profileConfig.HTTPSSPIFFE.EndpointSPIFFEID)
+		if err != nil {
+			return nil, fmt.Errorf("could not get endpoint SPIFFE ID: %w", err)
```

Comment:
suggest s/get/parse/ and to also log the spiffe id since it might not be clear on which federates_with we are erroring on (could also include td name or something if you want)

```suggestion
			return nil, fmt.Errorf("could not parse endpoint SPIFFE ID %q: %w", profileConfig.HTTPSSPIFFE.EndpointSPIFFEID, err)
```

Reply:
The caller to this function already includes the trust domain as part of the error description. This error is embedded in the error returned by the caller. E.g. the error will say (with your suggestion to include the SPIFFE ID): `error parsing federation relationship for trust domain "domain1.test": could not parse endpoint SPIFFE ID "spiffee://example.com/spire/server": spiffeid: invalid scheme`

Reply:
Ah great! Sorry I missed that